doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -80,10 +80,10 @@ For this tutorial, we'll be using the layer ``opengeo:countries`` to show off th
 
 Our layer is now ready to be served.
 
-Create OpenLayers application
------------------------------
+Create OpenLayers application - TMS Vector Tiles
+------------------------------------------------
 
-#. Create a ``www/vectortiles`` directory inside your GeoServer Data Directory.
+#. Create a ``www/tms-vectortiles`` directory inside your GeoServer Data Directory.
```

Comment:
Why this change?

Reply:
@bradh, this was to differentiate this example from the one I included. In case users want to follow both and have them in separated folders `tms-vectortiles` `wms-vectortiles`.